### DISCUSSION TITLE
SP7 touch screen multi touch

### DISCUSSION THREAD

**Discussion by squid-slime** [linux-surface/linux-surface#1506] [General]
i have been using this project for a few years on my sp7 so a massive thanks to the devs. I have noticed the two finger pinch works in firefox but things like 3 finger swipe does not. is this an issue with my DE?

**Comment by Flameopathic** [comment]
SP7 user, NixOS, Hyprland

i can get up to 10 touch points at a time, see https://cps-check.com/multi-touch-test
